includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -279,6 +285,7 @@ public function create_intention(
 	 * @param array   $level3                    - Level 3 data.
 	 * @param string  $selected_upe_payment_type - The name of the selected UPE payment type or empty string.
 	 * @param ?string $payment_country           - The payment two-letter iso country code or null.
+	 * @param string  $statement_descriptor       - (optional) The statement to be sent to the financial institution.
```

Comment:
Nit: The `(optional)` part makes me wonder if this is the only optional parameter, but it's not the case. I think we can dismiss the `(optional)` and change the type to `string|null`.

Also applies for `create_and_confirm_intention`.

Reply:
Applied in 106f141.